.github/workflows/go-develop.yml

```diff
@@ -175,18 +175,6 @@ jobs:
         run: |
           docker push -a ns1labs/orb-agent
 
-      - name: Build orb-ui
```

Comment:
no this is ok, we want these two in here for develop branch. we need to make sure it builds after a feature branch lands, and community members may use develop tag